_data/task.yml

```diff
@@ -60,23 +60,23 @@ websites:
     url: https://www.meistertask.com
     img: meistertask.png
     tfa:
-      - software
-    doc: https://support.meistertask.com/hc/en-us/articles/207318670
+      - totp
+    doc: https://support.mindmeister.com/hc/en-us/articles/218901367
 
   - name: Microsoft To-Do
     url: https://todo.microsoft.com/
     img: microsofttodo.png
     tfa:
       - sms
       - email
-      - software
+      - totp
```

Comment:
In additional to TOTP, Microsoft has their own authenticator app called [Microsoft Authenticator](https://play.google.com/store/apps/details?id=com.azure.authenticator&hl=en) so `proprietary` could also be included under supported 2FA options.

Reply:
The _Microsoft Authenticator_ app is covered by the `totp` method, since it is [RFC-6238](https://tools.ietf.org/html/rfc6238) compliant and can be replaced by other apps, such as _Google Authenticator_ or _andOTP_.

Reply:
Excellent. Thanks for the information.